### PR TITLE
Fix issues caused by latest CF AST changes

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

File: server/src/main/java/org/eclipse/lsp/cobol/core/model/extendedapi/Program.java [hunk 1]
```diff
@@ -14,13 +14,15 @@
...
 package org.eclipse.lsp.cobol.core.model.extendedapi;
 
+import lombok.EqualsAndHashCode;
 import lombok.Value;
 
 /**
- * Data transport object of CF AST. Program type is a root node of CF AST tree.
- * One file can contain more then one program.
+ * Data transport object of CF AST. Program type is a root node of CF AST tree. One file can contain
+ * more then one program.
```
Comment (asatklichov) [inline server/src/main/java/org/eclipse/lsp/cobol/core/model/extendedapi/Program.java:22]: than

**Review by asatklichov — APPROVED** [review]
🌊

File: server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java [hunk 2]
```diff
@@ -274,9 +282,17 @@ private void addParagraphRange(ParserRuleContext ctx) {
...
+    return getLocality(ctx.getStart())
+        .flatMap(
+            start ->
+                getLocality(ctx.getStop())
+                    .map(
+                        stop ->
+                            start.toBuilder()
+                                .range(
+                                    new Range(
+                                        start.getRange().getStart(), stop.getRange().getEnd()))
```
Comment (grianbrcom) [inline server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java:294]: The modern Art, I guess.

**Review by grianbrcom — APPROVED** [review]
Looks good, but I'm afraid that we will have a lot of issues related to `type` fields.

**Comment by temanbrcom** [comment]
> Looks good, but I'm afraid that we will have a lot of issues related to `type` fields.

I agree...

**Pull request merged**